### PR TITLE
Support for customer.DefaultAddress, customer.Addresses

### PR DESCRIPTION
**Pull request by seenickcode**
At least when we fetch a customer, the addresses can be populated properly. 

Later, I plan to submit create/update capability.

### CODE REVIEW

File: checkout.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	PrivacyPolicyURL    string        `json:"privacy_policy_url"`
+	RefundPolicyURL     string        `json:"refund_policy_url"`
+	TermsOfServiceURL   string        `json:"terms_of_service_url"`
+	UserID              int           `json:"user_id"`
+	WebURL              string        `json:"web_url"`
+	TaxLines            []struct {
+		Price string  `json:"price"`
+		Rate  float64 `json:"rate"`
+		Title string  `json:"title"`
+	} `json:"tax_lines"`
```
Comment (dlebech) [inline checkout.go:59]: Could you re-use the [`TaxLine`](https://github.com/getconversio/go-shopify/blob/9e3d52acdc5314e805f6399b908ec21413d68af3/order.go#L192-L196) struct here?

File: goshopify.go [hunk 2]
```diff
@@ -281,6 +285,9 @@ func CheckResponseError(r *http.Response) error {
 					topicAndElem := fmt.Sprintf("%v: %v", k, elem)
 					responseError.Errors = append(responseError.Errors, topicAndElem)
 				}
+			} else if reflect.TypeOf(v).Kind() == reflect.String {
+				topicAndElem := fmt.Sprintf("%v: %v", k, v)
+				responseError.Errors = append(responseError.Errors, topicAndElem)
```
Comment (dlebech) [inline goshopify.go:290]: What is this part? It looks like this is the reason Codecov is complaining about coverage. Could you add a test that covers this use case?

**Review by dlebech — CHANGES_REQUESTED** [review]
Thanks for the PR @seenickcode :100: and sorry for the delay in responding here.

Looks like good additions. I only have two comments, and then I think this is good to go :beers:

**Comment by seenickcode** [comment]
Hey thanks for the feedback. There'll be a delay in taking care of these due to some other blockers on another project I'm using this for.

**Comment by dlebech** [comment]
@seenickcode thanks for getting back to me. Everything sounds good. No rush :-)

**Comment by andrewhoff** [comment]
Hey @seenickcode we're going to close this PR. @rickywiens referenced it on the bold-commerce fork here https://github.com/bold-commerce/go-shopify/issues/3, where support for this project will continue. Feel free to open a PR there! Thanks!

**Pull request closed without merging**